beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/hive/HiveClient.java

```diff
@@ -100,4 +100,8 @@ public boolean tableExists(String databaseName, String tableName) {
           "Unexpected exception when checking if table \"" + databaseName + "." + tableName + "\" exists.", e);
     }
   }
+
+  public void close(){
```

Comment:
add implement Closeable to the class and you can do try with resources to auto close